ocaml/testsuite/tests/typing-tailcalls/basic.ml

```diff
@@ -0,0 +1,19 @@
+(* TEST
+ flags = "-dtypedtree -dsel";
```

Comment:
Please can we try to clean the output even further, so the test doesn't require promoting upon minor changes that cause identifier stamps etc to change?

Reply:
Done.

Reply:
> Please can we try to clean the output even further, so the test doesn't require promoting upon minor changes that cause identifier stamps etc to change?

Capturing `-dlambda` instead of `-dsel` now.